addons/medical_treatment/stringtable.xml

```diff
@@ -3496,5 +3496,13 @@
             <Chinesesimp>个人急救包使用条件</Chinesesimp>
             <Chinese>個人急救包使用條件</Chinese>
         </Key>
+        <Key ID="STR_ACE_Medical_Treatment_AllowSelfStitch_DisplayName">
+            <English>Self Stitching</English>
+            <French>Suturer soi-même</French>
+        </Key>
+        <Key ID="STR_ACE_Medical_Treatment_AllowSelfStitch_Description">
+            <English>Can the surgical kit be used on yourself?</English>
```

Comment:
Maybe: 

1. "Enables individuals to stitch themselves using a surgical kit."
2. "Enables the use of surgical kits to stitch oneself." (match the self IV setting)
3. "Controls whether individuals can stitch themselves."

Not sure which is best. With the medical settings cleanup, tried to avoid using questions as descriptions.

Reply:
I went with the second option, it's consistent with what we have.